02-peer-protocol.md

```diff
@@ -487,6 +487,9 @@ so we should fulfill the incoming HTLC as soon as we can to reduce latency.
 ### Risks With HTLC Timeouts
 
 
+Once an HTLC has timed out where it could either be fulfilled or timed-out;
```

Comment:
I think it refers to the fact that a-posteriori both states are possible, e.g., due to a late reveal of the payment key, and we must force a unique outcome by dropping to the chain early enough.